### PR TITLE
Update Travis config

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
+  global:
+    - CCACHE_DIR=$HOME/.ccache
   matrix:
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - USE_DEB=true ROS_DISTRO="indigo" ROS_REPO=ros NOT_TEST_INSTALL=true
```
Comment (mathias-luedtke) [inline .travis.yml]: `USE_DEB` is deprecated; in case of `=true`, you can safely remove it

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
-    - env: USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - env: USE_DEB=true ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=ros
+    - env: USE_DEB=true ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=ros-shadow-fixed
+
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config
 script: 
-  - source .ci_config/travis.sh
+- .ci_config/travis.sh
```
Comment (mathias-luedtke) [inline .travis.yml]: idention? works though..

File: .travis.yml (hunk 1 above)
Comment (gavanderhoorn) [reply .travis.yml]: Yeah, thanks. Just noticed it.

File: .travis.yml (hunk 2 above)
Comment (gavanderhoorn) [reply .travis.yml]: Not sure what happened there, this was a direct copy-paste. Fixed now.

**Comment by gavanderhoorn** [comment]
This reduces total Travis runtime for a single build from approx 1 hour to about 30 minutes (each job is around 6 to 7 minutes).

**Review by shaun-edwards — APPROVED** [review]

**Pull request merged**